primitives/linux-port/column.cpp

```diff
@@ -118,6 +119,24 @@ inline int compareBlock(const void* a, const void* b)
   return ((*(T*)a) - (*(T*)b));
 }
 
+template <class To, class From>
```

Comment:
I would rather put it into utils/common b/c this might be useful in many cases.

Reply:
Will do, thanks.